app/controllers/application_controller.rb

```diff
@@ -35,8 +35,8 @@ def render_unauthorized_not_conf_admin
     render_unauthorized(helpers.conf_admin?)
   end
 
-  def render_unauthorized(unauthorized)
-    return false if unauthorized
+  def render_unauthorized(authorized)
```

Comment:
pleeeeaaase use better naming and variables this is unnecessarily complicated :joy: